src/PerpetualPositionRouter.sol

```diff
@@ -36,6 +36,21 @@ contract PerpetualPositionRouter {
   // TODO: Should we deposit eth into the perpetual vault?
   receive() external payable {}
 
+  function combineArgs(uint8 funcName, uint160 sqrtPriceLimitX96) external pure returns (uint168) {
+    return (uint168(funcName) << 160) | uint168(sqrtPriceLimitX96);
+  }
+
+  function extractFuncName(uint168 args) internal pure returns (uint8) {
+    uint168 firstEightBitMask = ((1 << 8) - 1) << 160;
+    return uint8((args & firstEightBitMask) >> 160);
+  }
+
+  function extractSqrtPriceLimitX96(uint168 args) internal pure returns (uint160) {
+    uint168 mask = (1 << 160) - 1;
+
+    return uint160(args & mask);
+  }
```

Comment:
I will test these internal functions directly  when we refactor the tests to follow foundry best practices.

Reply:
#9

Reply:
Added this comment to the initial ticket